toolchain/check/testdata/class/adapter/adapt_copy.carbon

```diff
@@ -105,10 +105,9 @@ fn H(a: AdaptNoncopyableIndirect) -> AdaptNoncopyableIndirect {
 // CHECK:STDOUT: constants {
 // CHECK:STDOUT:   %AdaptCopyable: type = class_type @AdaptCopyable [template]
 // CHECK:STDOUT:   %int_32: Core.IntLiteral = int_value 32 [template]
-// CHECK:STDOUT:   %Int.type: type = fn_type @Int [template]
-// CHECK:STDOUT:   %Int: %Int.type = struct_value () [template]
-// CHECK:STDOUT:   %i32: type = int_type signed, %int_32 [template]
-// CHECK:STDOUT:   %complete_type: <witness> = complete_type_witness %i32 [template]
+// CHECK:STDOUT:   %i32: type = class_type @Int, @Int(%int_32) [template]
+// CHECK:STDOUT:   %i32.builtin: type = int_type signed, %int_32 [template]
+// CHECK:STDOUT:   %complete_type.2: <witness> = complete_type_witness %i32.builtin [template]
```

Comment:
Should this be attaching to `%i32` rather than `%i32.builtin`? Among other things, the comment on `CompleteTypeWitness` indicates that it's only created for class types.

Reply:
The operand of `complete_type_witness` is the object representation of the type whose completeness it represents, so this is indicating that `AdaptCopyable` (which uses this value) is complete and its object representation is the builtin 32-bit `IntType`.